src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -79,10 +76,10 @@
         <caption class="govuk-table__caption govuk-visually-hidden">Transfer tonnages for @aatf.AatfName</caption>
         <thead class="govuk-table__head">
             <tr class="govuk-table__row">
-                <th class="govuk-table__header govuk-!-text-align-left" style="width: 10%;">
+                <th class="govuk-table__header govuk-!-text-align-left" style="width: 15%;">
```

Comment:
can you not use any govuk class for setting the width, one-quarter and etc?

Reply:
The one-quarter rule seems to float left which ruins the formatting of the table
There are no set ones for 15%/35% and unsure if needed to create a CSS rule for this as unlikely to be used elsewhere

This specific formatting was taken from the Transfer Tonnages page